client/signup/main.jsx

```diff
@@ -124,11 +123,6 @@ const Signup = React.createClass( {
 		} );
 		recordSignupStart();
 
-		// Signup updates the cart through `SignupCart`. To prevent
-		// synchronization issues and unnecessary polling, the cart is disabled
-		// here.
-		disableCart();
```

Comment:
We still need to empty the cart here don't we?
In case we'll be reusing stale data from another signup or if a flow uses an existing site.